src/components/Result/Result.js

```diff
@@ -0,0 +1,19 @@
+function Result({ scoradResult }) {
+  let description;
```

Comment:
Stworzyłabym listę z tymi opisami i każdy opis byłby obiektem:
```suggestion
  const descriptions = [
  {
     description: "Twoje AZS ma łagodny przebieg",
     min: 0
     max: 14 
  },
  {...}
  ];
```